src/transform/ThemeTransform.js

```diff
@@ -41,14 +62,18 @@ const setTheme = (document, theme) => {
  * @return {void}
  */
 const classifyElements = element => {
-  Polyfill.querySelectorAll(element, 'img').forEach(image => {
-    if (!ElementUtilities.closestInlineStyle(image, 'background')) {
+  Polyfill.querySelectorAll(element, 'img')
+    .filter(imageNeedsWhiteBackground)
+    .forEach(image => {
       image.classList.add(CONSTRAINT.IMAGE_NO_BACKGROUND)
-    }
-    if (!ElementUtilities.isNestedInTable(image)) {
-      image.classList.add(CONSTRAINT.IMAGE_NONTABULAR)
-    }
-  })
+    })
+  /* en > Away colours > 793128975 */
```

Comment:
Oh I was using the same convention I used in the theme CSS file for consistency when referencing articles where issues occurred.